Documentation/Fr/Index.xyl

```diff
@@ -0,0 +1,744 @@
+  avec l'option <code>--filter</code> suivie par une <strong>expression</strong>.
+  Anecdote : Les expressions sont basées sur <a href="@hack:chapter=Ruler">la bibliothèque <code>Hoa\Ruler</code></a>!</p>
+  <p>Les variables suivantes sont disponibles dans l'<strong>expression</strong> du filtre :</p>
+  <ul>
+    <li><code>method</code> représente le nom du cas de test,</li>
+    <li><code>class</code> représente le nom de la suite de test,</li>
+    <li><code>namespace</code> représente l'espace de nom de la suite de test,</li>
+    <li><code>tags</code> représente la suite de test ou le label du cas de test.</li>
+  </ul>
+  <p>Tous les opérateurs standards de la bibliothèque <code>Hoa\Ruler</code>
+  peuvent être utilisés. Par exemple, pour lancer uniquement la suite de test
+  <code>Hoa\Foo\Test\Unit\Bar</code> qui est représentée par une classe, nous
+  écrirons :</p>
+  <pre><code clas="language-shell">$ vendor/bin/hoa test:run --directories Test/ --filter 'class = "Hoa\Foo\Test\Unit\Bar"'</code></pre>
+  <p>Ou, pour lancer uniquement deux cas de tests spécifiques, supposons
+  <code>case_sum</code> et <code>case_baz</code> :</p>
+  <pre><code clas="language-shell">$ vendor/bin/hoa test:run --directories Test/ --filter 'method = "case_sum" or method = "case_baz"'</code></pre>
+  <p>L'exemple précédent est strictement équivalent à :</p>
+  <pre><code clas="language-shell">$ vendor/bin/hoa test:run --directories Test/ --filter 'method in ["case_sum", "case_baz"]'</code></pre>
+  <p>Un autre exemple de filtre pour les suites de tests : Seulement ceux terminant par
+  <code>Qux</code> :</p>
+  <pre><code clas="language-shell">$ vendor/bin/hoa test:run --directories Test/ --filter 'class matches "Qux$"'</code></pre>
+  <p>L'opérateur <code>matches</code> nécessite une expression régulière respectant
+  <a href="http://pcre.org/">le format PCRE</a>.</p>
+  <p>Finalement, les cas de tests et les suites de tests peuvent avoir un ou plusieurs labels, merci à l'annotation
+  <code>@tags</code>. Une annotation est un commentaire du genre <code>/**
+  <em>…</em> */</code> localisé au dessus d'une classe, interface, méthode,
+  fonction etc. Les labels permettent d'avoir une classification des cas de
+  tests. Par exemple :</p>
+  <pre><code class="language-php">/**
+ * @tags featureA featureB
+ */
+public function case_sum()
+{
+    // …
+}</code></pre>
+  <p>Pour lancer uniquement les cas de tests avec le label <code>featureA</code>,
+  nous allons utiliser la ligne de commande suivante :</p>
+  <pre><code clas="language-shell">$ vendor/bin/hoa test:run --directories Test/ --filter '"featureA" in tags'</code></pre>
+  <p>Ici, la variable <code>tags</code> contient un tableau de chaines
+  représentant les noms des labels.</p>
+
+  <h3 id="Choose_the_PHP_virtual_machine" for="main-toc">Choose the PHP virtual
+  machine</h3>
+
+  <p>When testing, this is also useful to be able to select one specific
```

Comment:
@Hywan what about this section as we have dropped the support of HHVM ?

Reply:
HHVM still supports PHP for now, so let's keep it.